tasks/test-coverage.js

```diff
@@ -8,8 +8,8 @@
  */
 
 var fs = require('fs');
+var fse = require('fs-extra');
```

Comment:
you're right but we have the `fs` and `fse` vars in all the others scripts in the tasks folder.
we can change that in an other PR